release/long_running_tests/cluster.yaml

```diff
@@ -15,3 +15,10 @@ auth:
 
 head_node:
     InstanceType: m5.xlarge
+    TagSpecifications:
+        - ResourceType: "instance"
+          Tags:
+            - Key: anyscale-user
+              Value: "release-automation@anyscale.com"
+            - Key: anyscale-custodian
+              Value: "ignore"
```

Comment:
Should this be truly unlimited ?

Reply:
yeah we will have cost monitoring setup. and these are small instances